qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -682,6 +683,31 @@ def from_backend(
             channels=hamiltonian_channels,
         )
 
+        # Add control_channel_map from backend (only if not specified before by user)
+        if "control_channel_map" not in options:
+            control_channel_map = {}
+            if hasattr(backend, "control_channels"):  # Look in backend.control_channels
+                control_channel_map_backend = {**{qubits: backend.control_channels[qubits][0].index
+                                                  for qubits in backend.control_channels}
+                                               }
+            elif hasattr(backend, "coupling_map"):  # Look in backend.coupling_map
+                control_channel_map_backend = {
+                    **{qubits: backend.control_channel(qubits)[0].index for qubits in backend.coupling_map}
+                }
```

Comment:
If the labels can be completely arbitrary, Is there a way to reduce the control_channel_map dictionary to take into account only the qubits mentioned in the subsystem_list argument of the from_backend method? 
Otherwise my last block loading only the qubits in subsystem_list will not be general in that sense. Any idea how to frame it more generally?